sonar-plugin-api/src/main/java/org/sonar/api/batch/DecoratorBarriers.java

```diff
@@ -28,7 +28,9 @@
  * </ul>
  *
  * @since 2.3
+ * @deprecated since 5.6 no more used since Decorators are deprecated
```

Comment:
sentence is a bit strange. What about `@deprecated since 5.6 as Decorator is deprecated` ?